src/pages/Landing/index.tsx

```diff
@@ -56,6 +56,32 @@ export default () => {
   const columnHeaders = ['Name', 'Age', 'Is Cool'];
 
   if (isLoading) return <Loader />;
+  function handleDelete(customerId: any): void {
+    const deleteCustomer = () => {
+      const customers = customersGetter();
+      const index = customers.findIndex((customer) => customer.name === customerId);
+
+
+      if (index !== -1) {
+        const updatedCustomers = [...customers];
+        updatedCustomers.splice(index, 1);
+        customersSetter(updatedCustomers);
+      }
+    };
+
+    try {
+      deleteCustomer();
+      queryClient.invalidateQueries('customers');
+    } catch (error) {
+      console.error('An error occurred while deleting the customer:', error);
+
+    }
+  }
+  const actionItems = [
+    { label: 'Edit', onClick: (name) => handleEdit(name) },
+    { label: 'Delete', onClick: (name) => handleDelete(name) },
+  ];
```

Comment:
and here, you'd want to turn this into a function that returns the array of items, this function could take name as an argument and then these name variables would actually exist